src/components/chats/ChatRoom/ChatRoom.tsx

```diff
@@ -74,16 +74,12 @@ function ChatInputWrapper({
   customAction,
   scrollContainerRef,
 }: ChatInputWrapperProps) {
-  const clearReplyTo = useMessageData((state) => state.clearReplyTo)
-  const replyTo = useMessageData((state) => state.replyTo)
+  const clearAction = useMessageData((state) => state.clearAction)
   const sendEvent = useSendEvent()
 
   useEffect(() => {
-    return () => clearReplyTo()
-  }, [clearReplyTo])
-  const showEmptyPrimaryChatInput = useMessageData(
-    (state) => state.showEmptyPrimaryChatInput
-  )
+    return () => clearAction()
+  }, [clearAction])
```

Comment:
When the clearAction changing?

Idk when the useEffect should be triggered

Reply:
its cleanup function, triggered when ChatRoom is unmounted, I add clearAction because of rules of hooks warning
but in reality, it won't change, so it just runs when it unmounts